ports/fftw3/portfile.cmake

```
@@ -27,12 +28,17 @@ vcpkg_check_features(
         avx     ENABLE_AVX
         sse2    ENABLE_SSE2
         sse     ENABLE_SSE
+        neon    ENABLE_NEON
 )
 
 set(package_names  fftw3 fftw3f fftw3l)
-set(fftw3_options  "")
+if(VCPKG_TARGET_ARCHITECTURE STREQUAL "arm32")
+    set(fftw3_options "-DENABLE_NEON=OFF") # neon for double precision is not supported on arm32
+else()
+    set(fftw3_options  "")
+endif()

```

Let's keep it simple here:
```suggestion
set(fftw3_options  "")
```